angular/core/components/thread_page/activity_card/activity_card.coffee

```diff
@@ -56,7 +56,7 @@ angular.module('loomioApp').directive 'activityCard', ->
 
     $scope.threadItemVisible = (item) ->
       addSequenceId(item.sequenceId)
-      $scope.discussion.markAsRead(item.sequenceId)
+      item.markAsRead()
```

Comment:
This way we avoid the awkward process of taking an event, getting the discussion, and sending over its sequence id along with the discussion to the API.